Sources/Kommunicate/Classes/KMConversationViewController.swift

```diff
@@ -347,25 +347,40 @@ open class KMConversationViewController: ALKConversationViewController {
         NotificationCenter.default.post(notification)
     }
 
-    func updateAssigneeDetails() {
+    open override func updateAssigneeDetails() {
         conversationDetail.updatedAssigneeDetails(groupId: viewModel.channelKey, userId: viewModel.contactId) { contact, channel in
+            self.messageStatusAndFetchBotType()
             guard let alChannel = channel else {
                 print("Channel is nil in updatedAssigneeDetails")
                 return
             }
             self.customNavigationView.updateView(assignee: contact, channel: alChannel)
             self.assigneeUserId = contact?.userId
             self.hideInputBarIfAssignedToBot()
+            guard let contact = contact else {return}
+            self.isAwayMessageViewHidden = !contact.isInAwayMode
         }
     }
-
+    
+    /*
+     This method will verify status changed user id & current Conversation's assignee. If both are same then it will update.
+     - Parameters:
+     - userId: userId whose status changed
+     */
+    open override func updateAssigneeOnlineStatus(userId: String){
+        let (ConversationAssignee, _) = conversationDetail.conversationAssignee(groupId: viewModel.channelKey, userId: viewModel.contactId)
```

Comment:
If you are overriding, then call the super function as well. Same in the above function.